app-android/app/src/main/res/values-night-v31/styles.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+	<style name="SplashTheme" parent="Theme.AppCompat.Light.NoActionBar">
+		<item name="android:colorPrimary">@color/green</item>
+		<item name="android:colorPrimaryDark">@color/green</item>
+		<item name="android:windowBackground">@color/darkLighter</item>
+		<item name="android:statusBarColor">@color/darkLighter</item>
+		<item name="android:windowLightStatusBar">true</item>
```

Comment:
it still says `windowLightStatusBar` (in both)

Reply:
Fixed